internal/journald/journald.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package journald provides functions to read and collect journald logs.
+*/
+package journald
```

Comment:
Better name it `journalctl`, as this is the binary we are wrapping. Also, this can live inside bootstrapper/internal, as it is currently not shared between different microservices.